Sources/BitcoinKit/Core/Mnemonic/Mnemonic.swift

```diff
@@ -0,0 +1,256 @@
+//
```

Comment:
bah! I just made it more difficult to do a git diff, since I also moved this file (`Mnemonic.swift`) into a new group. Maybe you can do a manual diff?